bindings/python/crocoddyl/utils/copyable.hpp

```diff
@@ -56,12 +24,25 @@ struct CopyableVisitor : public bp::def_visitor<CopyableVisitor<C> > {
   template <class PyClass>
   void visit(PyClass& cl) const {
     cl.def("copy", &copy, bp::arg("self"), "Returns a copy of *this.");
-    cl.def("__copy__", &generic__copy__<C>);
-    cl.def("__deepcopy__", &generic__deepcopy__<C>);
+    cl.def("__copy__", &copy, bp::arg("self"), "Returns a copy of *this.");
+    cl.def("__deepcopy__", &deepcopy, bp::args("self", "memo"),
+           "Returns a deep copy of *this.");
+    cl.def("copy", &copy_ptr, bp::arg("self"), "Returns a copy of *this.");
+    cl.def("__copy__", &copy_ptr, bp::arg("self"), "Returns a copy of *this.");
+    cl.def("__deepcopy__", &deepcopy_ptr, bp::args("self", "memo"),
+           "Returns a deep copy of *this.");
```

Comment:
```suggestion
    cl.def("__copy__", &copy, bp::arg("self"), "Returns a copy of *this.");
    cl.def("__deepcopy__", &deepcopy, bp::args("self", "memo"),
           "Returns a deep copy of *this.");
```

Reply:
Closing this thread as it belongs to discussion in https://github.com/loco-3d/crocoddyl/pull/1151#discussion_r1278619301.